README.md

```diff
@@ -32,13 +32,14 @@ Smartstack relied on Zookeeper as a key-value store to keep track of all the dis
 You can start use your own Zookeeper, or simply start it in container (and expose it to the host on port 2181):
 
 ```
-docker run -d -p 2181:2181 jplock/zookeeper
+CID=$(docker run -d -p 2181:2181 jplock/zookeeper)
+docker inspect $CID
 ```
 
 ### Start the Color UI Service
 
 ```
-docker run -d -e ZK_HOSTS=192.168.59.103:2181 -p 80:80 pierrevincent/discovery-demo-color-ui-service
+docker run -d -e ZK_HOSTS=<ZK_IPADDRESS>:2181 -p 80:80 pierrevincent/discovery-demo-color-ui-service
```

Comment:
The ZK_IPADDRESS can be "localhost" in a Linux environment as the zookeeper container is exposing 2181 to the host (-p 2181:2181), so it's not necessary to figure out exactly which IP the container is running on.